addons/mail/models/mail_render_mixin.py

```diff
@@ -435,12 +441,15 @@ def _render_template(self, template_src, model, res_ids, engine='inline_template
         :param dict add_context: additional context to give to renderer. It
           allows to add or update values to base rendering context generated
           by ``MailRenderMixin._render_<engine>_eval_context()``;
-        :param dict options: options for rendering;
-        :param boolean post_process: perform a post processing on rendered result
-          (notably html links management). See``_render_template_postprocess``;
+        :param dict options: options for rendering; may contain
+            boolean post_process: perform a post processing on rendered result
+            (notably html links management). See``_render_template_postprocess``);
 
         :return dict: {res_id: string of rendered template based on record}
         """
+        if options is None:
+            options = {}
```

Comment:
```suggestion
        options = options or {}
```

Reply:
Actually doing that you create a dict when you give a void dict which is useless. I think this is more an old habit of mine, either you give None (default) and I instantiate a new dict, either I use the parameter given. If you give False that's your fault.